demo/src/screens/nativeComponentScreens/keyboardAccessory/KeyboardAccessoryViewScreen.js

```diff
@@ -170,7 +172,7 @@ export default class KeyboardInputViewScreen extends PureComponent {
   };
 
   render() {
-    const {message, isModal} = this.props;
+    const {message/* , isModal */} = this.props;
```

Comment:
Why not remove the `isModal` altogether? Where should it come from? Or is it an example?

Reply:
It is an example. I want to keep it